### ISSUE TITLE
Write tests for service navigation_service.dart

### ISSUE THREAD

**Issue by palisadoes** [PalisadoesFoundation/talawa#2158]
The Talawa code base needs to be 100% reliable. This means we need to have 100% unittest code coverage.

We will need unittests done for all methods, classes and functions found in this file.

Any widgets referenced in this file must also have unittests done

```
lib/services/navigation_service.dart
```

The current code coverage for this file can be found here:
-  https://app.codecov.io/gh/PalisadoesFoundation/talawa?displayType=list

**Comment by Manik2708** [comment]
@palisadoes @noman2002 
Can you assign this issue to me

**Comment by noman2002** [comment]
@Manik2708 Assigned to you, please consider adding a profile picture. It helps us to remember people who are consistently working.

**Comment by Manik2708** [comment]
PR #2230 raised, please see for the review!

**Comment by palisadoes** [comment]
This issue has to be reopened. The PR is causing our workflows to fail.

**Comment by Manik2708** [comment]
> This issue has to be reopened. The PR is causing our workflows to fail.

Can you tell me reason of failing of Workflows?

**Comment by Manik2708** [comment]
> > This issue has to be reopened. The PR is causing our workflows to fail.
> 
> Can you tell me reason of failing of Workflows?

After carefully watching the order of PRs, PR #2231 was raised just after #2230 but was merged before the PR #2230 and PR #2231 modifies the same change in `navigation_service.dart` which leads to Errors. Will be raising a new PR shortly!

**Comment by Manik2708** [comment]
PR #2240 raised, Kindly see for review!

**Comment by noman2002** [comment]
> > > This issue has to be reopened. The PR is causing our workflows to fail.
> > 
> > 
> > Can you tell me reason of failing of Workflows?
> 
> After carefully watching the order of PRs, PR #2231 was raised just after #2230 but was merged before the PR #2230 and PR #2231 modifies the same change in `navigation_service.dart` which leads to Errors. Will be raising a new PR shortly!

@Manik2708 PRs are not merged in the order. Any PR which got the approval can be merged

**Comment by noman2002** [comment]
> PR #2240 raised, Kindly see for review!

Sure, thanks.

**Comment by Manik2708** [comment]
> > > > This issue has to be reopened. The PR is causing our workflows to fail.
> > > 
> > > 
> > > Can you tell me reason of failing of Workflows?
> > 
> > 
> > After carefully watching the order of PRs, PR #2231 was raised just after #2230 but was merged before the PR #2230 and PR #2231 modifies the same change in `navigation_service.dart` which leads to Errors. Will be raising a new PR shortly!
> 
> @Manik2708 PRs are not merged in the order. Any PR which got the approval can be merged

Not complaining, was just laughing at the situation, how the same file was edited by two PRs at the same time :)

**Comment by noman2002** [comment]
Please be more objective in your tone. And this is no where written that same file cannot be edited at once. Both the issues are completely different. One deals with test coverage while the other fixes a bug.

**Comment by Wreck-X** [comment]
> > > > > This issue has to be reopened. The PR is causing our workflows to fail.
> > > > 
> > > > 
> > > > Can you tell me reason of failing of Workflows?
> > > 
> > > 
> > > After carefully watching the order of PRs, PR #2231 was raised just after #2230 but was merged before the PR #2230 and PR #2231 modifies the same change in `navigation_service.dart` which leads to Errors. Will be raising a new PR shortly!
> > 
> > 
> > @Manik2708 PRs are not merged in the order. Any PR which got the approval can be merged
> 
> Not complaining, was just laughing at the situation, how the same file was edited by two PRs at the same time :)

oh lol might've been my bad the bug fix was required for my pr to work ill communicate better next time :D

**Comment by Manik2708** [comment]
> > > > > > This issue has to be reopened. The PR is causing our workflows to fail.
> > > > > 
> > > > > 
> > > > > Can you tell me reason of failing of Workflows?
> > > > 
> > > > 
> > > > After carefully watching the order of PRs, PR #2231 was raised just after #2230 but was merged before the PR #2230 and PR #2231 modifies the same change in `navigation_service.dart` which leads to Errors. Will be raising a new PR shortly!
> > > 
> > > 
> > > @Manik2708 PRs are not merged in the order. Any PR which got the approval can be merged
> > 
> > 
> > Not complaining, was just laughing at the situation, how the same file was edited by two PRs at the same time :)
> 
> oh lol might've been my bad, the bug fix was required for my pr to work ill communicate better next time :D

No worries! Thanks for your concern.
